src/config.cc

```diff
@@ -2,9 +2,12 @@
 
 #include <boost/algorithm/string/classification.hpp>
 #include <boost/algorithm/string/split.hpp>
+#include <sstream>
 
 #include "logger.h"
 
+using std::string;
+
```

Comment:
Using "using namespace" considered as bad practice. I prefer to remove it and add std to all std variables.